### PR TITLE
Laura/issue40 link accessibility

### PR DESCRIPTION
**Pull request by LauraLangdon**
I think I've now optimized every link on the site for accessibility to the best of my knowledge, so closing Issue #40.

### CODE REVIEW

File: website/docs/compute/integrate-the-function-editor/code-editor.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ The Compute code editor is available for you to embed in your application, so yo
 
 The editor is hosted at `https://editor.suborbital.network`, and uses URL parameters to configure its connection to your builder service.
 
-You will host the Compute [builder service](../concepts/data-plane-vs-control-plane.md) in your cloud infrastructure with a configured DNS name. In order to use the editor, an HTTPS connection to the builder is required.
+You will host the Compute builder service](../concepts/data-plane-vs-control-plane.md) in your cloud infrastructure with a configured DNS name. In order to use the editor, an HTTPS connection to the builder is required.
```
Comment (jagger27) [inline website/docs/compute/integrate-the-function-editor/code-editor.md:9]:
```suggestion
[You will host the Compute builder service](../concepts/data-plane-vs-control-plane.md) in your cloud infrastructure with a configured DNS name. In order to use the editor, an HTTPS connection to the builder is required.
```

File: website/docs/compute/quickstart/create-an-environment-token-with-subo.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ brew install subo
 
 Then, run `subo --version` to ensure the installation was successful. **The `version` must be at least `v0.2.1` for full Compute support.**
 
-To install on Linux (or macOS without Homebrew), you can download [Subo directly](https://github.com/suborbital/subo/releases).
+To install on Linux (or macOS without Homebrew), you can [download Subo directly](https://github.com/suborbital/subo/releases).
 
 ## Generate your token
-You can get a Compute Environment token either with our [web app](https://suborbital.network/) or with the `subo` command line tool.
+You can get a Compute Environment token either with our [environment token generator web app](https://suborbital.network/) or with the `subo` command line tool.
```
Comment (jagger27) [inline website/docs/compute/quickstart/create-an-environment-token-with-subo.md:19]: I thought we weren’t going to advertise this anymore

**Review by jagger27 — APPROVED** [review]
Small broken link/typo, otherwise looks good to me!

File: website/docs/compute/quickstart/client-libraries/golang.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ go get github.com/suborbital/compute-go@latest
 
 ## Basic Usage
 ### Configuration
-This example sets up a basic client with the token generated in [Step 1](../1.-create-an-environment-token-with-subo.md) or with the web app. The `compute.Client` object created here assumes that Compute is running on the same host on its default ports. The code for this example can be found [here](https://github.com/suborbital/compute-go/blob/main/examples/app).
+This example sets up a basic client with the token generated in [Step 1](../1.-create-an-environment-token-with-subo.md) or with the web app. The `compute.Client` object created here assumes that Compute is running on the same host on its default ports. Feel free to [check out the code for this example!](https://github.com/suborbital/compute-go/blob/main/examples/app)
```
Comment (javorszky) [inline website/docs/compute/quickstart/client-libraries/golang.md:15]:
```suggestion
This example sets up a basic client with the [token generated in Step 1](../1.-create-an-environment-token-with-subo.md) or with the web app. The `compute.Client` object created here assumes that Compute is running on the same host on its default ports. Feel free to [check out the code for this example!](https://github.com/suborbital/compute-go/blob/main/examples/app)
```

Comment (javorszky) [reply website/docs/compute/quickstart/client-libraries/golang.md:15]: Expanded the link text

**Review by javorszky — CHANGES_REQUESTED** [review]
Minor typos, but otherwise excellent!

File: website/docs/compute/quickstart/create-an-environment-token-with-subo.md (hunk 2 above)
Comment (LauraLangdon) [reply website/docs/compute/quickstart/create-an-environment-token-with-subo.md:19]: Per @cohix on Discord, we'll keep the link.

**Pull request merged**